assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -193,9 +203,11 @@ export default function EffectChooser({
     const listWithDisabledOptions = [...BACKGROUND_EFFECTS_LIST].filter(
       // eslint-disable-next-line consistent-return
       (currentEffect) => {
-        const isDisabled = disabledTypeOptionsMap[
-          'effect-background-pan'
-        ].find((disabledOption) => currentEffect.endsWith(disabledOption));
+        const isDisabled =
+          currentEffect &&
+          disabledTypeOptionsMap[
+            'effect-background-pan'
+          ].find((disabledOption) => currentEffect?.endsWith(disabledOption));
```

Comment:
this is a preemptive bug fix - found it just now. If you have an image in the foreground, animate it, open the animation dropdown chooser again and then click the 'set as background image' button with the drop down open and then open the animation dropdown chooser again we'd see that the current value that the dropdown has initially is false and filtering for proper available background animations doesn't like it.